spec/rails_helper.rb

```diff
@@ -12,6 +12,8 @@
 # use for there expectations, etc.
 ENV['IIIF_HOST_URL'] = 'localhost'
 
+ENV['USE_JWT_AUTH'] = 'Y'
```

Comment:
Should this be in individual tests? I was thinking that we could check to make sure that when the USE_JWT_AUTH is set to 'N' (or boolean), that the will not be hit.